Source/WTF/wtf/win/RunLoopWin.cpp

```diff
@@ -58,18 +58,10 @@ LRESULT RunLoop::wndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
         performWork();
         return 0;
     case SetTimerMessage:
-        ::SetTimer(hWnd, wParam, lParam, nullptr);
         return 0;
-    case KillTimerMessage:
-        ::KillTimer(hWnd, wParam);
-        return 0;
-    case WM_TIMER:
+    case FireTimerMessage:
         RunLoop::TimerBase* timer = nullptr;
-        {
-            Locker locker { m_loopLock };
-            if (m_liveTimers.contains(wParam))
-                timer = std::bit_cast<RunLoop::TimerBase*>(wParam);
-        }
+        timer = std::bit_cast<RunLoop::TimerBase*>(wParam);
```

Comment:
`timer` can be already destroyed here. I fixed it in https://github.com/WebKit/WebKit/pull/43244